tests/flows/steps/test_setup_parameters.py

```diff
@@ -0,0 +1,65 @@
+from swo_aws_extension.flows.order import InitialAWSContext
+from swo_aws_extension.flows.steps import SetParametersVisibleStep
+from swo_aws_extension.parameters import (
+    PARAM_PHASE_ORDERING,
+    OrderParametersEnum,
+    get_ordering_parameter,
+)
+
+
+def test_update_parameter_visibility(order_factory):
```

Comment:
I think would great to minimize the interface of the `Step`s and always test `__call__` method. WDYT?